packages/components/src/panel/test/body.js

```diff
@@ -1,97 +1,126 @@
 
-		it( 'should change state and class when sidebar is closed', () => {
-			const panelBody = shallow(
-				<PanelBody title="Some Text" initialOpen={ false } />
+		it( 'should render inner content, if opened', () => {
+			const { container } = render(
+				<PanelBody opened={ true }>
+					<div className="inner-content">Content</div>
+				</PanelBody>
 			);
-			expect( panelBody.state( 'opened' ) ).toBe( false );
-			expect( panelBody.hasClass( 'is-opened' ) ).toBe( false );
+			const panelContent = getPanelBodyContent( container );
+
+			expect( panelContent ).toBeTruthy();
 		} );
 
-		it( 'should use the "opened" prop instead of state if provided', () => {
-			const panelBody = shallow(
-				<PanelBody
-					title="Some Text"
-					opened={ true }
-					initialOpen={ false }
-				/>
+		it( 'should be opened by default', () => {
+			const { container } = render(
+				<PanelBody>
+					<div>Content</div>
+				</PanelBody>
 			);
-			expect( panelBody.state( 'opened' ) ).toBe( false );
-			expect( panelBody.hasClass( 'is-opened' ) ).toBe( true );
-		} );
+			const panelContent = getPanelBodyContent( container );
 
-		it( 'should render child elements within PanelBody element', () => {
-			const panelBody = shallow( <PanelBody children="Some Text" /> );
-			expect( panelBody.instance().props.children ).toBe( 'Some Text' );
-			expect( panelBody.text() ).toBe( 'Some Text' );
+			expect( panelContent ).toBeTruthy();
 		} );
 
-		it( 'should pass children prop but not render when sidebar is closed', () => {
-			const panelBody = shallow(
-				<PanelBody children="Some Text" initialOpen={ false } />
+		it( 'should render as initially opened, if specified', () => {
+			const { container } = render(
+				<PanelBody initialOpen={ true }>
+					<div>Content</div>
+				</PanelBody>
 			);
-			expect( panelBody.instance().props.children ).toBe( 'Some Text' );
-			// Text should be empty even though props.children is set.
-			expect( panelBody.text() ).toBe( '' );
+			const panelContent = getPanelBodyContent( container );
+
+			expect( panelContent ).toBeTruthy();
 		} );
 	} );
 
-	describe( 'mounting behavior', () => {
-		it( 'should mount with a default of being opened', () => {
-			const panelBody = mount( <PanelBody /> );
-			expect( panelBody.state( 'opened' ) ).toBe( true );
-		} );
+	describe( 'toggling', () => {
+		it( 'should toggle collapse with opened prop', () => {
+			const { container, rerender } = render(
+				<PanelBody opened={ true }>
+					<div>Content</div>
+				</PanelBody>
+			);
+			let panelContent = getPanelBodyContent( container );
 
-		it( 'should mount with a state of not opened when initialOpen set to false', () => {
-			const panelBody = mount( <PanelBody initialOpen={ false } /> );
-			expect( panelBody.state( 'opened' ) ).toBe( false );
-		} );
-	} );
+			expect( panelContent ).toBeTruthy();
+
+			act( () => {
+				rerender(
+					<PanelBody opened={ false }>
+						<div>Content</div>
+					</PanelBody>
+				);
+			} );
```

Comment:
I'm pretty sure you don't need to wrap `rerender` calls in `act`. It seems that it's already called in `react-testing-library`: https://github.com/testing-library/react-testing-library/blob/9e5cf59ceaa66b534ef95d519cacb8afc4814215/src/pure.js#L60-L66

Reply:
@diegohaz Thank you!! 🙏 . Made the updates!